library/create_kafka.py

```diff
@@ -0,0 +1,239 @@
+#!/usr/bin/python
+
+
+from __future__ import (absolute_import, division, print_function)
+from sys import api_version
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+---
+module: create_kafka
+
+short_description: Create Red Hat OpenShift Streams for Apache Kafka Instance
+
+# If this is part of a collection, you need to use semantic versioning,
+# i.e. the version is of the form "2.5.0" and not "2.4".
+version_added: "0.1.0"
+
+description: Create Red Hat OpenShift Streams for Apache Kafka Instance
+
+options:
+    name:
+        description: Name of the Kafka instance
+        required: true
+        type: str
+    cloud_provider:
+        description: Cloud provider for the Kafka instance
+        required: false
+        type: str
+        default: aws
+    region:
+        description: Region of the Kafka instance
+        required: false
+        type: str
+        default: eu-west-1
+    reauthentication_enabled:
+        description: Reauthentication enabled for the Kafka instance
+        required: false
+        type: bool
+        default: true
+    plan:
+        description: Plan for the Kafka instance
+        required: false
+        type: str
+        default: ""
+    billing_cloud_account_id:
+        description: Billing cloud account id for the Kafka instance
+        required: false
+        type: str
+        default: ""
+    marketplace:
+        description: Marketplace for the Kafka instance
+        required: false
+        type: str
+        default: ""
+    billing_model:
+        description: Billing model for the Kafka instance
+        required: false
+        type: str
+        default: marketplace
+    instance_type:
+        description: Instance type for the Kafka instance
+        required: false
+        type: str
+        default: developer
+ 
+# Specify this value according to your collection
+# in format of namespace.collection.doc_fragment_name
+extends_documentation_fragment:
+    - my_namespace.my_collection.my_doc_fragment_name
+
+author:
+    - Red Hat Developer
+'''
+
+EXAMPLES = r'''
+# Pass in a message
+  - name: Create kafka
+    create_kafka:
+      name: "struttin"
+    register:
+      kafka_req_resp 
+
+ 
+'''
+
+RETURN = r'''
+# These are examples of possible return values, and in general should use other names for return values.
+original_message:
+    description: The original kafka request payload that was passed in.
+    type: dict
+    returned: always
+    sample: "invocation": {
```

Comment:
Values from mock. Not up to date already hahaha. Lets not include this. 
I do not see why this would be useful for anyone as text. 
Link to SDK would be much better.

Reply:
👍